docs/coding-guidelines/adding-api-guidelines.md

```diff
@@ -43,7 +43,7 @@ the implementation without compat concerns in future releases.
 ## Making the changes in repo
 
 **If changing the library version**
-  - Update the `AssemblyVersion` property in `<Library>\Directory.Build.props` (ex: [Microsoft.CSharp\Directory.Build.props](https://github.com/dotnet/runtime/blob/master/src/libraries/Microsoft.CSharp/Directory.Build.props#L4)) to the version determined above.
+- The `AssemblyVersion` property isn't always in `<Library>\Directory.Build.props` file unless we want to override the default for some reason. For more information on how the original version is calculated, see [Arcade documentation](https://github.com/dotnet/arcade/blob/master/Documentation/CorePackages/Versioning.md).
```

Comment:
I think the comment is fine, but it is probably worth mentioning as well (given that the title of the section is if changing the library version) how to change the assembly version. That is, to mention that if there is an assemblyversion property on the Directory.Build.props of the library you can just rev up that one, and if there isn't one, you can just add it over there (and you could also just add an example of a library that already does it). Does that make sense?

Reply:
@joperezr Hope it's good now?

Reply:
Perfect 😄